healthid/build.gradle.kts

```diff
@@ -38,8 +38,8 @@ publishing {
         withType<MavenPublication> {
             artifact(javadocJar)
             pom {
-                name.set("ePrescription OTT")
-                description.set("Use data representing OTT for ePrescription as a SD JWT Credential")
+                name.set("Health ID Attestation")
```

Comment:
do we also want change coordinates and publish relocation POM on maven central, so updates work even for transitive dependencies between old and new coordinates?

Reply:
That would be great